### PR TITLE
[UPDATE] to SIRIUS 4.8.2

### LINKED ISSUE #5337: Update to SIRIUS 4.8.2
Update THIRDPARTY and check tests.

### PR DESCRIPTION
**Pull request by oliveralka**
# Description

Update to SIRIUS 4.8.2
fixes #5337

Some of the tests changed mostly due to score changes (different rank of analytes) & additional/altered databases.

# Checklist:
- [x] Make sure that you are listed in the AUTHORS file
- [x] Add relevant changes and new features to the CHANGELOG file
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] New and existing unit tests pass locally with my changes
- [x] Updated or added python bindings for changed or new classes. (Tick if no updates were necessary.)

# How can I get additional information on failed tests during CI:
If your PR is failing you can check out 
- http://cdash.openms.de/index.php?project=OpenMS and look for your PR. If you click in the column that lists the failed tests you will get detailed error messages.

### CODE REVIEW

**Comment by oliveralka** [comment]
If everything works, THIRPARTY has to be merged first and updated here (https://github.com/OpenMS/THIRDPARTY/pull/74).

**Comment by oliveralka** [comment]
rebuild jenkins

File: src/tests/topp/THIRDPARTY/SiriusAdapter_9_output.mzTab [hunk 1]
```diff
@@ -1,14 +1,14 @@
 MTD	mzTab-version	1.0.0
 MTD	mzTab-mode	null
 MTD	mzTab-type	null
-MTD	description	Sirius-4.4.29
+MTD	description	Sirius-4.6.0
```
Comment (oliveralka) [inline src/tests/topp/THIRDPARTY/SiriusAdapter_9_output.mzTab:4]: When pushing THIRDPARTY, change Sirius/CSI writer class to version 4.8.2.

**Comment by oliveralka** [comment]
rebuild jenkins

**Review by jpfeuffer — APPROVED** [review]
At some point, one could parse the version from the command-line help.

**Comment by oliveralka** [comment]
@jpfeuffer Is there a way to restart the deepcode-ci-bot?

**Comment by jpfeuffer** [comment]
Probably. But you can ignore it for this PR anyway.

**Comment by jpfeuffer** [comment]
If you want this to work with our conda package, one should upgrade there, too.

**Comment by oliveralka** [comment]
@jpfeuffer Can I find a short instruction, how to do that somewhere?

**Comment by jpfeuffer** [comment]
No the thing is, Sirius must be upgraded first. https://bioconda.github.io/recipes/sirius-csifingerid/README.html?highlight=sirius#package-package%20&#x27;sirius-csifingerid&#x27;

Since it is Java, usually you just need to update the link to the "package" and update some copy statements.

**Pull request merged**